### ISSUE TITLE
[request] does not work with 1.20.1

### ISSUE THREAD

**Issue by TwoPointInfinity** [wikmor/LPC#32]
the most recent version is not compatible with 1.20.1

**Comment by Risusama** [comment]
Any idea what kind of errors etc. show up, if any?

**Comment by Risusama** [comment]
@TwoPointInfinity I am running this on 1.20.1 Paper, works totally fine. Just because it is not listed as "compatible" to it, doesn't always mean it won't work. One should always test a plugin first, before assuming it won't work and reporting back "it doesn't work", even though it does.

**Comment by TwoPointInfinity** [comment]
> @TwoPointInfinity I am running this on 1.20.1 Paper, works totally fine. Just because it is not listed as "compatible" to it, doesn't always mean it won't work. One should always test a plugin first, before assuming it won't work and reporting back "it doesn't work", even though it does.

Risusama,
I want to address your comment directly. I did test LPC on 1.20.1, and it didn't work on my server at all. It's not a matter of assuming compatibility without testing. My report was meant to highlight a legitimate issue, not to jump to conclusions. Your assumption that I didn't test it comes across as disrespectful. Everything in your comment after "works totally fine" was unnecessary and belittling. A better was of saying this could have been by asking "LPC works for me on 1.20.1, did you test it yet or have you not tried because it doesn't list 1.20.1 as a compatible version?" Let's keep the conversation focused on resolving this compatibility matter.

**Comment by Risusama** [comment]
> > @TwoPointInfinity I am running this on 1.20.1 Paper, works totally fine. Just because it is not listed as "compatible" to it, doesn't always mean it won't work. One should always test a plugin first, before assuming it won't work and reporting back "it doesn't work", even though it does.
> 
> Risusama, I want to address your comment directly. I did test LPC on 1.20.1, and it didn't work on my server at all. It's not a matter of assuming compatibility without testing. My report was meant to highlight a legitimate issue, not to jump to conclusions. Your assumption that I didn't test it comes across as disrespectful. Everything in your comment after "works totally fine" was unnecessary and belittling. A better was of saying this could have been by asking "LPC works for me on 1.20.1, did you test it yet or have you not tried because it doesn't list 1.20.1 as a compatible version?" Let's keep the conversation focused on resolving this compatibility matter.

Yeah, I could have worded it better. I didn't mean to sound rude, I was just writing it on mobile and on the go, so I didn't really spend like, any time on the wording - so I do indeed need to apologize for that. Sorry. 🙏  But, you simply said "not compatible with 1.20.1", with no other information. What server did you use? Were there any errors, warnings, etc. anything?
